client/my-sites/domains/domain-management/contacts-privacy/card.jsx

```diff
@@ -110,29 +137,49 @@ class ContactsPrivacyCard extends React.PureComponent {
 			);
 		}
 
+		let privacyText = translate(
+			'{{strong}}Privacy Protection{{/strong}} is turned {{strong}}off{{/strong}} for this domain. ' +
+				'Your contact information is {{strong}}public{{/strong}}.',
+			{
+				components: {
+					strong: <strong />,
+				},
+			}
+		);
+
+		if ( privateDomain ) {
+			privacyText = translate(
+				'{{strong}}Privacy Protection{{/strong}} is turned {{strong}}on{{/strong}} for this domain. ' +
+					'Your contact information is {{strong}}private{{/strong}}. ',
+				{
+					components: {
+						strong: <strong />,
+					},
+				}
+			);
+		}
+
 		return (
-			<Notice status="is-warning" showDismiss={ false }>
-				{ translate(
-					'{{strong}}Privacy Protection{{/strong}} is turned off for this domain. ' +
-						'Your contact information is {{strong}}public{{/strong}}. ' +
-						'{{a}}Enable Privacy Protection{{/a}}',
-					{
-						components: {
-							strong: <strong />,
-							a: (
-								<a
-									href={ domainManagementPrivacyProtection(
-										selectedSite.slug,
-										selectedDomainName
-									) }
-								/>
-							),
-						},
-					}
-				) }
-			</Notice>
+			<div className="contacts-privacy__info">
+				<p>{ privacyText }</p>
+				<Button
+					className="contacts-privacy__info-button"
+					disabled={ submitting }
+					busy={ submitting }
+					onClick={ this.togglePrivacy }
+				>
+					{ privateDomain ? translate( 'Disable Privacy' ) : translate( 'Enable Privacy' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Disable Whois privacy.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).